ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -395,8 +398,14 @@ objects:
         sed -i '/^  channel:/d' /tmp/artifacts/installer/manifests/cvo-overrides.yaml
 
         if [[ "${KURYR_ENABLED:-false}" != "true" ]]; then
-          echo "Forcing OpenShiftSDN by modifying manifests"
-          sed -i -e 's/networkType: .*$/networkType: OpenShiftSDN/g' /tmp/artifacts/installer/manifests/cluster-network-02-config.yml
+          if [[ "${CLUSTER_VARIANT:-sdn}" == "sdn" ]]; then
```

Comment:
the historic `CLUSTER_VARIANT` takes a range of comma-delimited values, so it's good to use something like [`has_variant`][1].  But it would be even better to move these tests into [the multi-step framework][2].

[1]: https://github.com/openshift/release/blob/0178d76d26e927c620cdf072cdf9283d0a79c067/ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml#L413-L419
[2]: https://github.com/openshift/release/tree/0178d76d26e927c620cdf072cdf9283d0a79c067/ci-operator/step-registry